ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/hardware_counters.schema.yml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  hardware_counters:
+    type: list
```

Comment:
This should be a dict, with features as the only key.

Reply:
Corrected!
 This one is in draft due to convert_types issue for list

Reply:
added validator support

Reply:
Thanks